### PR TITLE
Some more polishing I should have done earlier (orthography, formatting)

### PR DESCRIPTION
**Pull request by vasyugan**

### CODE REVIEW

**Comment by vasyugan** [comment]
Work some more on the German translation before I make another request

**Comment by vasyugan** [comment]
No, need some sleep now. Therefore reopening pull request...

File: user_manual_de/pim/sync_thunderbird.rst [hunk 1]
```diff
@@ -1,29 +1,46 @@
...
-   -  Im "**URL**"-Feld tragen Sie die URL ein, die Ihnen in der Nextcloud "Kontakte"-App nach Klick auf das Zahnrad-Symbol angezeigt wird. Dieses Symbol ist im unteren linken Bereich der "Kontakte"-App zu finden.
-   - Wenn Sie die Adressen nur in eine Richtigung (von der Nextcloud zu Thunderbird, nicht aber andersherum) synchronisieren möchten, aktivieren Sie "Nur lesen"
-   - Bestätigen Sie mit "OK"
 
-Das Verschieben eines Kontaktes kann ungewollte Folgen haben. Wenn Sie einen Kontakt beispielsweise via Drag & Drop
-aus Ihrem Nextcloud-Adressbuch in das lokale Adressbuch verschieben, wird der Kontakt von Ihrer Nextcloud gelöscht
-und ist dann nur noch lokal verfügbar. Sie können Ihre Kontakte als VCF-Datei mit der Nextcloud Web-Oberfläche oder
-als LDIF per Thunderbird Adressbuch sichern.
+Alternative: Die Erweiterung CardBook (Nur Kontakte)
+------------------------------------------------------
```
Comment (skjnldsv) [inline user_manual_de/pim/sync_thunderbird.rst]:
```suggestion
----------------------------------------------------
```

File: user_manual_de/pim/sync_thunderbird.rst [hunk 2]
```diff
@@ -1,29 +1,46 @@
-Thunderbird - Adressbuch synchronisieren
-========================================
+==============================
+Synchronisation mit Thunderbird
+==============================
```
Comment (skjnldsv) [inline user_manual_de/pim/sync_thunderbird.rst]:
```suggestion
===============================
```

File: user_manual_de/pim/sync_thunderbird.rst [hunk 3]
```diff
@@ -1,29 +1,46 @@
-Thunderbird - Adressbuch synchronisieren
-========================================
+==============================
```
Comment (skjnldsv) [inline user_manual_de/pim/sync_thunderbird.rst]:
```suggestion
===============================
```

**Comment by vasyugan** [comment]
> See comments :)
> 
> I added you to the org, so you can also create branch on the documentation.
> That way tests can be run against your work. Would you mind doing this? :)
> Create a new pr with those changes directly on https://github.com/nextcloud/documentation?

Whatever you say. But I am not sure what the steps involved are. I am a bloody beginner at github. So please bear with me...

**Comment by vasyugan** [comment]
I am a bit confused now. Dispite you accepting this pull request, https://github.com/nextcloud/documentation/blob/master/user_manual/pim/sync_thunderbird.rst is still different from https://github.com/vasyugan/documentation/blob/master/user_manual/pim/sync_thunderbird.rst What did I miss?

**Comment by skjnldsv** [comment]
> Dispite you accepting this pull request

it is not accepted, it needs to be merged for it to be fully integrated :)

**Comment by vasyugan** [comment]
> > Dispite you accepting this pull request
> 
> it is not accepted, it needs to be merged for it to be fully integrated :)

Alright. Sorry for the confusion. Still having a hard time understanding how Github works. So am I expected to do something at this point or just wait?

**Comment by skjnldsv** [comment]
> Whatever you say. But I am not sure what the steps involved are. I am a bloody beginner at github. So please bear with me...

No worries! :hugs: 

> Alright. Sorry for the confusion. Still having a hard time understanding how Github works. So am I expected to do something at this point or just wait?

Just wait a bit :)
I'm not fond of merging stuff without our tests passing. But until we fixed our automated setup, it's not going to be easy ;)

Next time you can just edit directly from the documentation, no need to use your fork (https://github.com/nextcloud/documentation/ versus https://github.com/vasyugan/documentation)

**Comment by vasyugan** [comment]
> > Whatever you say. But I am not sure what the steps involved are. I am a bloody beginner at github. So please bear with me...
> 
> No worries! hugs
> 
> > Alright. Sorry for the confusion. Still having a hard time understanding how Github works. So am I expected to do something at this point or just wait?
> 
> Just wait a bit :)
> I'm not fond of merging stuff without our tests passing. But until we fixed our automated setup, it's not going to be easy ;)
> 
> Next time you can just edit directly from the documentation, no need to use your fork (https://github.com/nextcloud/documentation/ versus https://github.com/vasyugan/documentation)

So as soon as my changes have been merged, can I safely delete my fork?

**Comment by skjnldsv** [comment]
> So as soon as my changes have been merged, can I safely delete my fork?

yes :)

File: user_manual_de/pim/sync_thunderbird.rst [hunk 4]
```diff
@@ -1,29 +1,45 @@
-Thunderbird - Adressbuch synchronisieren
-========================================
+===============================
+Synchronisation mit Thunderbird
+==============================
```
Comment (skjnldsv) [inline user_manual_de/pim/sync_thunderbird.rst]:
```suggestion
===============================
```

Comment (skjnldsv) [reply user_manual_de/pim/sync_thunderbird.rst]: @vasyugan it is still missing one char length :)

Comment (vasyugan) [reply user_manual_de/pim/sync_thunderbird.rst]: Hm, it seems the pull request is still open. I have accepted all your proposed changes. So what is holding it up?

Comment (skjnldsv) [reply user_manual_de/pim/sync_thunderbird.rst]: The one above i still not fixed :)

**Comment by vasyugan** [comment]
Am 30.01.20 um 09:25 schrieb John Molakvoæ:
>
> *@skjnldsv* commented on this pull request.
>
> ------------------------------------------------------------------------
>
> In user_manual_de/pim/sync_thunderbird.rst
> <https://github.com/nextcloud/documentation/pull/1797#discussion_r372811630>:
>
> > @@ -1,29 +1,45 @@
> -Thunderbird - Adressbuch synchronisieren
> -========================================
> +===============================
> +Synchronisation mit Thunderbird
> +==============================
>
> The one above i still not fixed :)
>
Better now?

**Pull request merged**